shared/scss/_mixins.scss

```diff
@@ -223,3 +223,21 @@
         }
     }
 }
+
+@mixin placeholder {
+    .is-browser--chrome & {
+        &::-webkit-input-placeholder {@content}
+    }
+    .is-browser--moz & {
+        &:-moz-placeholder {@content}
+        &::-moz-placeholder {@content}
+    }
+}
+
+@mixin arrow-left($size, $color) {
```

Comment:
I thought I'd leave the name as generic as possible even though it's meant mainly for popouts